### ISSUE TITLE
BUG Phpunit Conflict in Version 1.0.8

### ISSUE THREAD

**Issue by 5now** [symfony/symfony#41610]
When I'm going to start my test in symfony 5.1 with PHPUnit/PHPUnit.xml 8.2.5 it will crash because in the composer.json
of this package required phpunit = "*".

 PHP Fatal error:  Uncaught Error: Call to undefined method PHPUnit\TextUI\TestRunner::doRun() in /var/www/html/current/bin/.phpunit/phpunit-8.2-0/src/TextUI/Command.php:201
Stack trace:
#0 /var/www/html/current/bin/.phpunit/phpunit-8.2-0/src/TextUI/Command.php(160): PHPUnit\TextUI\Command->run(Array, true)
symfony/test-pack#1 /var/www/html/current/bin/.phpunit/phpunit-8.2-0/phpunit(22): PHPUnit\TextUI\Command::main()
symfony/test-pack#2 /var/www/html/current/vendor/symfony/phpunit-bridge/bin/simple-phpunit.php(429): include('/var/www/html/c...')
symfony/test-pack#3 /var/www/html/current/bin/phpunit(13): require('/var/www/html/c...')
symfony/test-pack#4 {main}
  thrown in /var/www/html/current/bin/.phpunit/phpunit-8.2-0/src/TextUI/Command.php on line 201
make: *** [Makefile:304: app-functionaltest] Fehler 255

**Comment by nicolas-grekas** [comment]
> composer unpack symfony/test-pack

that's the solution! packs should always be unpacked now. They are if you're using a recent flex version. If not, upgrade! and unpack.

**Comment by dklymenk** [comment]
For me the fix was to just upgrade to 9.5.

`phpunit.xml.dist`:
```xml
...
    <server name="SYMFONY_PHPUNIT_VERSION" value="9.5"/>
...
```

**Comment by 5now** [comment]
For me also after a couple of hours to changing my tests for in my eye's not necesarry change in the code.
